test/integration/safer_cluster/controls/network.rb

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+project_id   = attribute('project_id')
+cluster_name = attribute('cluster_name')
+
+control "network" do
+  title "gcp network configuration"
+  describe google_compute_firewalls(project: project_id) do
+    its('firewall_names') { should include "gke-#{cluster_name[0..25]}-intra-cluster-egress" }
```

Comment:
Not a ruby expert but looking at this [1] wont we need to discard the last char as we need length to be at most 25chars? So `[0...25]` or `[0,25]`
[1]: https://stackoverflow.com/questions/9690801/difference-between-double-dot-and-triple-dot-in-range-generation

Reply:
@Dev25  Saw this error in CI
Cluster name: `safer-cluster-cluster-7yid`
Actual FW name when I checked: `gke-safer-cluster-cluster-7yi-intra-cluster-egress`
What we checked for in the test: `gke-safer-cluster-cluster-7yid-intra-cluster-egress`
```
  ×  network: gcp network configuration (2 failed)
     ×  google_compute_firewalls firewall_names should include "gke-safer-cluster-cluster-7yid-intra-cluster-egress"
     expected ["default-allow-icmp", "default-allow-internal", "default-allow-rdp", "default-allow-ssh", "gke-deplo...node-http-hc", "k8s-fw-a2ee97657700f434d9340944b00a5b61", "k8s-fw-ae0f40471c88f42199bb5196a6394f69"] to include "gke-safer-cluster-cluster-7yid-intra-cluster-egress"
     ×  google_compute_firewalls firewall_names should include "gke-safer-cluster-cluster-7yid-webhooks"
     expected ["default-allow-icmp", "default-allow-internal", "default-allow-rdp", "default-allow-ssh", "gke-deplo...node-http-hc", "k8s-fw-a2ee97657700f434d9340944b00a5b61", "k8s-fw-ae0f40471c88f42199bb5196a6394f69"] to include "gke-safer-cluster-cluster-7yid-webhooks"
```

Reply:
Changed to `[0,25]`, thanks for helping debug this.